10up/classifai
# Add: Ability to process already created content

### Description of the Change
Add bulk and row actions to classify supported post types.
Add a new class: `Classifai\Admin\BulkActions` to manage bulk actions, not only
for posts classification, but also images.
Use the `SavePostHandler::classify()` method to classify posts.


### Alternate Designs
n/a


### Benefits
Provide UI for classifying supported post types in bulk.


### Possible Drawbacks
n/a


### Verification Process

1. Allow post/page to be classified.
2. Go to Posts list table.
3. Select some posts (Choose posts that haven't classified yet).
4. Choose `Classify` from bulk actions select box.
5. Click Apply.
6. See posts classified and a success notice.



### Checklist:






### Applicable Issues
Closes #121. Relate: #129.
